README.md

```diff
@@ -2,41 +2,57 @@
 
 This package is a [release-it](https://github.com/release-it/release-it) plugin
 (using [`release-it`'s plugin
-API](https://github.com/release-it/release-it/tree/master/docs/plugins)) that
-reads `workspaces` configuration from your `package.json` and bumps each
-individual package's version and publishes.
+API](https://github.com/release-it/release-it/blob/master/docs/plugins.md)) that
+releases each of your projects configured workspaces.
+
+## How it works
+
+In order to publish each of your projects workspaces, we first check the root
+`package.json` to determine the locations of each of your workspaces (handling
+both globbing and various formats for `workspaces`). Once we have identified
+all of the workspaces, we bump the `package.json`s `version` field to the
+selected version and publish the package (by changing into the package's root
+folder and calling `npm publish`).
 
 ## Usage
 
 Installation using your projects normal package manager, for example:
 
-```
+```sh
 # npm
 npm install --save-dev release-it-yarn-workspaces
 
-# yarn add --dev release-it-yarn-workspaces
+# yarn
+yarn add --dev release-it-yarn-workspaces
```

Comment:
```suggestion
yarn add --dev -W release-it-yarn-workspaces
```

I think we should add the `-W`, since that's the flag that instructs workspaces to install to the root package.json, and since they're using a workspaces repo this is likely what they want.